js/my.js

```diff
@@ -28,6 +28,7 @@
 
 var clients = {};
 var privMSG = {};
+var chans = [];
```

Comment:
I'm not sure what chans is used for here.  We should probably be putting these in privMSG.